README.md

```diff
@@ -150,35 +152,132 @@ and if it is disabled (boolean).
 
 ### Test Helper
 
-Since `x-select` uses internal identifiers as the `value` attribute, it
-doesn't integrate with the `fillIn` test helper. But don't fret, we've built a
-test helper for you.
+`<XSelect>` 4.0 ships with an entirely new test helper that goes
+beyond just allowing you to select an option. It allows you to
+interact with your `<select>` element in all different ways. For
+example in tests if you need to assert your first options `disabled`
+or not:
+
+```javascript
+expect(xselect.options(0).isDisabled).to.equal(true);
+```
+
+Under the hood this new test helper is using a [BigTest
+Interactor](https://bigtestjs.io/guides/interactors/introduction/). Interactors
+allow you to think about how you're going to _interact_ with the DOM
+and abstract that into composable & immutable containers. Interactors
+are similar to page objects, but for components.
 
 #### Using the test helper
 
-To use the select helper in your tests you have to import the select function:
+Import the select interactor:
 
 ``` javascript
-  import { select } from "yourappname/tests/helpers/x-select";
+// you can name the import whatever you want
+import XSelect from "yourappname/tests/helpers/x-select";
```

Comment:
Just for the sake of the rest of the examples here, I think it should be `XSelectInteractor`

Reply:
Hahah that's funny, I was trying to hand wave over the fact that we're using an interactor and make it just about the helper. But I agree, there's no escaping it really.